### PR TITLE
Adds Elite Security (was a prank, memes over now)

### PR DESCRIPTION
**Pull request by F-OS**
as you can see i am not trying to merge this to the april fools branch, this is not a joke, and before you say Muh balance Muh Immulsions hear me out, traitors wizards nuke ops and other antags are far from balanced, double esword traitor can wipe the crew out so i think sec should be allowed to have a elite response unit to murderboning wiznerds and traitors, this is my first pr so feedback is appreciated, and thanks for the idea and code Bawhoppen. Also Mapfreeze so this cant be merged til that's over

### CODE REVIEW

**Comment by Zilenan91** [comment]
>gave them a pulse carbine and a laser cannon in a belt

This IS an april fools joke right

**Comment by F-OS** [comment]
@Zilenan91 i will replace the laser cannon with a teargas grenade

**Comment by Bawhoppen** [comment]
the greatest april fools PR of all time

**Comment by F-OS** [comment]
@Zilenan91 and no, this is not a april fools joke

**Comment by Bawhoppen** [comment]
ok well if you're serious that it isn't, then you're retarded

but if it is, then this is a genius play, cause we all got so caught up in our april fools content we forgot we could still get april fools pranked

**Comment by Aloraydrel** [comment]
>Antags are far from balance
That is because antags are against the entire station hence why they get semi overpowered items

**Comment by F-OS** [comment]
@Bawhoppen how am i retarded?

**Comment by Bawhoppen** [comment]
I'm not getting caught in this trap

**Comment by Bawhoppen** [comment]
also if you want actual criticism here you go:

first off, you need to move the full military belt to the actual belts file
second off, some of that job code is messed up and needs to be fixed, I just copy pasted cause it was only meant for april fools day
third off, they don't have any actual landmark spawnpoints, and you can't add any to the maps because of dunc's mapfreeze
fourth off, military belts can't hold everything I put into them, it was part of the joke
fifth off, am I getting trapped
sixth off, balance meme

**Comment by F-OS** [comment]
@Bawhoppen thanks for the criticisim, i will do all of that!

**Comment by F-OS** [comment]
also what would be required to make this balanced? i am kinda new to coding

**Comment by Core0verload** [comment]
\>kinda new to coding
\>uses old changelog system
Not falling for it, not even for a second.

**Comment by Bawhoppen** [comment]
@Core0verload its not hard to check their past contributions, they've done shit before

File: code/modules/jobs/job_types/security.dm [hunk 1]
```diff
@@ -322,3 +322,35 @@ var/list/sec_departments = list("engineering", "supply", "medical", "science")
...
+			            access_forensics_lockers, access_morgue, access_maint_tunnels, access_medical)
+	
+/datum/outfit/job/elo
+	name = "Elite Security Officer"
+	belt = /obj/item/weapon/storage/belt/military/full
+	ears = /obj/item/device/radio/headset/headset_sec/alt
+	uniform = /obj/item/clothing/under/rank/security/navyblue
+	shoes = /obj/item/clothing/shoes/combat/swat
+	suit = /obj/item/clothing/suit/armor/laserproof
+	suit_store = /obj/item/weapon/gun/energy/pulse/carbine
```
Comment (KazeEspada) [inline code/modules/jobs/job_types/security.dm]: YOU CRAZY? A FUCKING PULSE RIFLE?

**Comment by Steelpoint** [comment]
I can't tell if this is serious or not.

**Comment by Archie700** [comment]
Either way the joke is on us

**Comment by Rodocks** [comment]
Reee donut kno how to nut get raped gimme pulse rifles pls ; (

**Comment by PKPenguin321** [comment]
this is the best april fools joke of all time

**Comment by Bawhoppen** [comment]
@PKPenguin321 excuse me that comment violates copyright please remove

File: code/modules/jobs/job_types/security.dm (hunk 1 above)
Comment (F-OS) [inline code/modules/jobs/job_types/security.dm]: you crazy, giving traitors eswords, ebows and holoparasites!! balance traitors and you will have a case. either way this job only has 2 slots open and there are normally only 4 traitors

Comment (PKPenguin321) [inline code/modules/jobs/job_types/security.dm]: somebody's salty lmao

Comment (Bawhoppen) [inline code/modules/jobs/job_types/security.dm]: you overdid that reply, now its blatant this is a joke


try again next year

**Comment by F-OS** [comment]
welp, it was a fun prank while it lasted, and it was a april fools joke, thanks for the laughs everyone, cheers!

**Comment by PKPenguin321** [comment]
i knew ittt :laughing: good shit fam

**Comment by Zilenan91** [comment]
just a prank bro

**Comment by F-OS** [comment]
ok, i fixed the bug that caused it to not compile, opening and closing to force travis build

**Pull request closed without merging**